examples/string.cpp

```diff
@@ -14,7 +14,6 @@
  */
 
 #include <aws/cryptosdk/cpp/kms_keyring.h>
-#include <aws/cryptosdk/default_cmm.h>
```

Comment:
Since the contents of `default_cmm.h` is now a private implementation detail we should consider moving this header file from `include/default_cmm.h` to `include/private/default_cmm.h`.

Reply:
See above comment. This is not a private implementation detail. Default CMMs are still part of the public API, but fewer people need to use them.

Reply:
Gotcha!